server/methods/accounts/accounts.js

```diff
@@ -356,6 +356,7 @@ export function addressBookAdd(address, accountUserId) {
           "profile.addressBook.$.isShippingDefault": false
         }
       });
+      // should the userId passed to this hook match the userId of the updated Account?
```

Comment:
These seem like unrelated comments

Reply:
it is definitely unrelated. I noticed it when trying to stub some data. I guess it would have been better as a PR comment than a code comment

Reply:
but is is a bug, right?
this method allows for a userId to be passed in, but several of the operations take place on the _current_ user.